src/service/cid-impl.js

```diff
@@ -213,10 +213,16 @@ export class Cid {
     const scope = getCidStruct.scope;
     /** @const {!Location} */
     const url = parseUrl(this.ampdoc.win.location.href);
+    let api = null;
     if (!isProxyOrigin(url)) {
+      api = this.cidApi_;
+    } else if (this.cacheCidApi_.isSupported()) {
+      api = this.cacheCidApi_;
+    }
+    if (api) {
       const apiKey = this.isScopeOptedIn_(scope);
       if (apiKey) {
```

Comment:
If the scope is not opted in the CID API, then it falls back to using a cookie? Is that correct on the AMP Cache?